docs/data/about-the-data.md

```diff
@@ -11,41 +11,41 @@ This section describes in detail exactly what data exists in Event Data, and wha
+Because Event Data captures links between 'traditional literature' and 'non-traditional literature', the 'subject' of most Events is a webpage and the 'object' is a registered content item, linked via its DOI.
 
-Crossref members already provide reference information for Registered Content, for example article-to-article citations. This falls under the 'traditional scholarship' category and is outside the scope of Event Data. If the Publisher has made it open, this data is available via the [Crossref REST API](https://api.crossref.org).
+Crossref members already provide reference information for registered content, for example article-to-article citations. This falls under the 'traditional scholarship' category and is outside the scope of Event Data. If the publisher has made it open, this data is available via the [Crossref REST API](https://api.crossref.org).
 
-"Webpages" represent a wide variety of things, including members' sites. In order to avoid accidentally collecting 'traditional' reference information, all Crossref Event Data Agents avoid websites that belong to our members. We maintain a list of domains that the Event Data Agents will not visit because we know they belong to Publishers. This exclusion is done on a best-effort basis.
+"Webpages" represent a wide variety of things, including members' sites. In order to avoid accidentally collecting 'traditional' reference information, all Crossref Event Data Agents avoid websites that belong to our members. We maintain a list of domains that the Event Data Agents will not visit because we know they belong to publishers. This exclusion is done on a best-effort basis.
 
 We also exclude other webpages that we believe are part of databases, library systems etc. As we monitor a potentially limitless set of websites, this with is done with heuristics on a best-effort basis.
 
 <a name="data-sources"></a>
 
 ## Sources and Agents
 
-Event Data is a hub for the collection and distribution of a variety of Events and contains data from a selection of Sources. Every Event is produced by an Agent operated by Crossref, DataCite or one of our partners. The Agent is responsible for connecting to an external Data Source and turning the data into Events. In some cases, the Agent is passing data directly through and in other cases the Agent has to do some work to extract the Event.
+Event Data is a hub for the collection and distribution of a variety of Events and contains data from a selection of sources. Every Event is produced by an Agent operated by Crossref, DataCite or one of our partners. The Agent is responsible for connecting to an external data source and turning the data into Events. In some cases, the Agent is passing data directly through and in other cases the Agent has to do some work to extract the Event.
```

Comment:
There's a difference between 'source' in a general sense and 'Source' as a specific entity in Event Data. I think we should keep upper case.